app/src/main/res/layout/toolbar_session.xml

```diff
@@ -27,9 +24,26 @@
             app:layout_constraintBottom_toBottomOf="parent"
             app:layout_constraintEnd_toEndOf="parent"
             app:layout_constraintHorizontal_bias="0.0"
+            app:layout_constraintVertical_bias="0.1"
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toTopOf="parent"
-            app:layout_constraintVertical_bias="0.95"
+            tools:text="My Notes" />
```

Comment:
Please use meaningful texts on "tools:text" property to help other developers understand what is going to be the real text.